kube/client.go

```diff
@@ -63,7 +63,7 @@ type ClientInterface interface {
 	GetNamespaceStatus(namespace string) (AutomaticDeploymentOption, error)
 	GetNamespaceUserSecretName(namespace, username string) (string, error)
 	GetUserDataFromSecret(namespace, secret string) (string, string, error)
-	CreateTempConfig(namespace, serviceAccount string) (string, string, error)
+	GetSAToken(namespace, serviceAccount string) (string, error)
```

Comment:
I know this is consistent with other funcs, but its not Go-like to use `get` prefix for functions.